CMakeLists.txt

```diff
@@ -16,17 +16,18 @@ ament_auto_add_library(laser_scan_filters SHARED src/laser_scan_filters.cpp)
 ament_auto_add_library(laser_filter_chains SHARED
                          src/scan_to_cloud_filter_chain.cpp
                          src/scan_to_scan_filter_chain.cpp)
-rclcpp_components_register_nodes(laser_filter_chains
-                         "ScanToCloudFilterChain"
-                         "ScanToScanFilterChain")
 
-set(FILTER_CHAINS
-    scan_to_cloud_filter_chain
-    scan_to_scan_filter_chain
```

Comment:
Could you add a comment describing what rclcpp_components_register_node does here? It's a very cool and useful macro, but the documentation isn't super easy to find, and it'd be nice to have a summary here.

Reply:
I added a comment that briefly describes the macro